webpack.config.js

```diff
@@ -21,11 +21,11 @@ module.exports = {
   module: {
     rules: [{
       test: path.resolve(__dirname, 'presets', 'core.ts'),
-      use: { loader: 'expose-loader', options: 'sfxCore' }
+      use: { loader: 'expose-loader', options: 'GbElementsCore' }
```

Comment:
If we're keeping the same convention as before, this new object should be called `gbElementsCore` instead of `If we're keeping the same convention as before, this new object should be called `GbElementsCore`. Is there any reason for the new capitalization?

(I noticed this because I'm updating the demo in the View Layer and needed to instantiate Core)

Reply:
I never starting the names of objects used as "variable bags" with a lowercase letter. It's not really a value that you'd pass around and use by itself: you'd always access some property of it. We have `Math`, `JSON`, `Symbol`, and [others](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Not_a_constructor) as precedents in favour of capitalizing the first letter.